src/components/LoansByCategory/HelpmeChoose/HelpmeChooseRecommendations.vue

```diff
@@ -61,16 +43,31 @@ export default {
 		HelpmeChooseBorrowerSelector,
 		KvMaterialIcon
 	},
+	props: {
+		loans: {
+			type: Array,
+			default: () => []
+		},
+	},
 	data() {
 		return {
 			mdiRefresh,
-			selectedLoan: 0
+			selectedLoan: 0,
 		};
 	},
+	computed: {
+		loanData() {
```

Comment:
Looks like this isn't being used currently.

Reply:
Yeah, this data will be injected to a loan card in other ticket.